bottomless/src/replicator.rs

```diff
@@ -935,6 +935,12 @@ impl Replicator {
         })?;
         tracing::trace!("Local change counter: {change_counter}");
 
+        // TODO: we shouldn't leak the connection here but for some reason when this connection get
+        // dropped it seems to checkpoint the database
+        if std::env::var("LIBSQL_DISABLE_INIT_CHECKPOINTING").is_ok() {
```

Comment:
Let's maybe call it `LIBSQL_DISABLE_BOTTOMLESS_CHECKPOINTING` ? With this change it does not affect only init checkpointing, does it?

Reply:
Only reason I didn't call it that was because it felt better to have a single env var for this change and some of the code we feature flagged was in sqld.

Reply:
Ok I gave the bottomless code its own feature flag with `LIBSQL_BOTTOMLESS_`, this fits the style we are already using in bottomless.